service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,157 @@
+        </description>
+        <out-parameters>
+            <parameter name="orderItemDetails"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="orderItems" list="orderItemDetails">
+                <econdition field-name="orderId" value="100102"/>
+                <econdition field-name="orderPartSeqId" value="01"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderParts">
+       <description>
+           Find all Order Parts with partTotal less than equals to 20.
+       </description>
+        <out-parameters>
+            <parameter name="orderPartDetails"/>
+        </out-parameters>
+        <actions>
+           <entity-find entity-name="orderParts" list="orderPartDetails">
+               <econdition field-name="partTotal"  operator="less-equals" value="20"/>
+           </entity-find>
+        </actions>
+    </service>
+    
+    <service verb="get" noun="OrderPartData">
+        <description>
+            Find all Order Part records assigned to the facility with ZIRET_WH value
+        </description>
+        <out-parameters>
+            <parameter name="orderPartDetails" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="orderParts" list="orderList">
+                <econdition field-name="facilityId"  value="ZIRET_WH"/>
+             </entity-find>
+            <set field="orderPartDetails" value="[]"/>
+            <iterate list="orderList" entry="part">
+                <set field="orderId" from="part.orderId"/>
+                <set field="facilityId" from="part.facilityId"/>
+                <set field="partMap" from="[orderId:orderId,facilityId:facilityId]"/>
+                <set field="orderPartDetails" from="orderPartDetails+partMap"/>
+            </iterate>
+        </actions>
+    </service>
+    
+    <service verb="get" noun="OrderPartsCount">
+        <description>
+            Find the count of order parts for the customer Party Id, “CustJqp”
+        </description>
+        <out-parameters>
+            <parameter name="orderPartsCount"/>
+        </out-parameters>
+        <actions>
+            <entity-find-count entity-name="orderParts" count-field="orderPartsCount">
+                <econdition field-name="customerPartyId" operator="equals" value="CustJqp"/>
+            </entity-find-count>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderPartsInfo">
+        <description>
+            Find all unique Order Parts with shipmentMethodEnumId value 
+            as “ShtMthGround” and facilityId as “ZIRET_WH”
+        </description>
+        <out-parameters>
+            <parameter name="orderPartsList"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="orderParts" list="orderPartsList">
+                <econdition field-name="shipmentMethodEnumId" operator="equals" value="ShtMthGround"/>
```

Comment:
Can you check if there are multiple order parts with the same value of shipmentMethodEnumId and facilityId will it return unique records or return all as per this code?

Reply:
Okay, Ma'am. I'll check and resolve it.